### PR TITLE
IGAPP-1001: Use the target name as product name

### PR DESCRIPTION
**Pull request by maxammann**
This pull request belongs to an issue on our [bugtracker](https://issues.integreat-app.de/).
You can find it there by looking for an issue with the key which is mentioned in the title of this pull request.
It starts with the keyword **IGAPP**.

### CODE REVIEW

**Review by SteffiStuffel — APPROVED** [review]
Approval with no idea whats happening here.

File: native/ios/Integreat.xcodeproj/project.pbxproj [hunk 1]
```diff
@@ -740,7 +739,7 @@
 				ENABLE_NS_ASSERTIONS = NO;
 				ENABLE_STRICT_OBJC_MSGSEND = YES;
 				ENABLE_TESTABILITY = YES;
-				"EXCLUDED_ARCHS[sdk=iphonesimulator*]" = "arm64 ";
+				"EXCLUDED_ARCHS[sdk=iphonesimulator*]" = "";
```
Comment (klinzo) [inline native/ios/Integreat.xcodeproj/project.pbxproj:742]: Is this intended? If so, why?

File: native/ios/Integreat.xcodeproj/project.pbxproj [hunk 2]
```diff
@@ -758,7 +757,6 @@
 				LIBRARY_SEARCH_PATHS = (
 					"\"$(SDKROOT)/usr/lib/swift\"",
 					"$(TOOLCHAIN_DIR)/usr/lib/swift/$(PLATFORM_NAME)",
-					"$(TOOLCHAIN_DIR)/usr/lib/swift-5.0/$(PLATFORM_NAME)",
```
Comment (klinzo) [inline native/ios/Integreat.xcodeproj/project.pbxproj:761]: Is this intended? If so, why?

**Review by f1sh1918 — APPROVED** [review]
works fine on macOS, no changes anymore when starting xcode

**Comment by maxammann** [comment]
> works fine on macOS, no changes anymore when starting xcode

I reverted all weird changes. So if there are changes with XCode I think we should open a separate task, or just commit it with the next PR.

**Pull request merged**